tools/expressionmerge.py

```diff
@@ -35,7 +36,7 @@
     base, ext = os.path.splitext(f)
     delimiter = ';' if ext == '.csv' else '\t'
 
-    with gzip.open(f, 'rb') as csvfile:
+    with io.TextIOWrapper(io.BufferedReader(gzip.open(f))) as csvfile:
```

Comment:
Very interesting. Yes, we should definitely introduce a utility function (`utils.gzopen`) and use it everywhere. Indeed, `io.BufferedReader` looks very nice performance wise.